tests/upgrade/migration/migrator.go

```diff
@@ -0,0 +1,227 @@
+package migration
+
+import (
+	cheqdkeeper "github.com/cheqd/cheqd-node/x/cheqd/keeper"
+	cheqdtestssetup "github.com/cheqd/cheqd-node/x/cheqd/tests/setup"
+	cheqdtypes "github.com/cheqd/cheqd-node/x/cheqd/types"
+	resourcekeeper "github.com/cheqd/cheqd-node/x/resource/keeper"
+	resourcetestssetup "github.com/cheqd/cheqd-node/x/resource/tests/setup"
+	resourcetypes "github.com/cheqd/cheqd-node/x/resource/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type MigrationScenario struct {
```

Comment:
Needed, but not in its current form: this will just become invoke previous syntax and then invoke existing tests.